examples/component-actions/zarf.yaml

```
@@ -39,6 +39,21 @@ components:
         # runs after the component is deployed
         after:
           - cmd: touch test-create-after.txt
+          - cmd: sleep 1
+          - cmd: echo "I can print!"
+          - cmd: sleep 1
+          - cmd: |
+              echo "multiline!"
+              sleep 1
+              echo "updates!"
+              sleep 3
+              echo "in!"
+              sleep 1
+              echo "realtime!"
+              sleep 1
+            description: multiline & description demo
+          - cmd: sleep 1
+          - cmd: for i in $(seq 1 10); do echo "The current number is $i. I can loop and evaluate expressions too!"; done
```

```suggestion
            description: multiline & description demo
```
^ We dont need this and could remove it if it is causing issues.